internal/network/manifests_generator.go

```diff
@@ -68,6 +69,81 @@ spec:
   osImageURL: ""
 `
 
+const snoDnsmasqConf = `
+address=/{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/{{.HOST_IP}}
+addn-hosts=/etc/api-int.host
+`
+
+const apiIntHosts = `
+{{.HOST_IP}} api-int api-int.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}
```

Comment:
Why not have a single DNS entry `*.clusterName.baseDomain` pointing to the host IP?

Reply:
Your question is more like , why we need this file right? will check

Reply:
It will be different from the convention we have right now, asked about it and sounds like better to stick to the current one where we have apps. ... and api-int as independent